devicetypes/heltun/heltun-rs01-switch.src/heltun-rs01-switch.groovy

```diff
@@ -0,0 +1,571 @@
+     name: "Group 1", options: [
+                0: "Disabled",
+                12: "1st & 2nd Relay",
+                13: "1st & 3rd Relay",
+                14: "1st & 4th Relay",
+                15: "1st & 5th Relay",
+                23: "2nd & 3rd Relay",
+                24: "2nd & 4th Relay",
+                25: "2nd & 5th Relay",
+                34: "3rd & 4th Relay",
+                35: "3rd & 5th Relay",
+                45: "4th & 5th Relay"
+        ], paramNum: 101, size: 1, default: "0", type: "enum"],
+
+    [name: "Group 2", options: [
+                0: "Disabled",
+                12: "1st & 2nd Relay",
+                13: "1st & 3rd Relay",
+                14: "1st & 4th Relay",
+                15: "1st & 5th Relay",
+                23: "2nd & 3rd Relay",
+                24: "2nd & 4th Relay",
+                25: "2nd & 5th Relay",
+                34: "3rd & 4th Relay",
+                35: "3rd & 5th Relay",
+                45: "4th & 5th Relay"
+        ], paramNum: 102, size: 1, default: "0", type: "enum"],
+
+    [title: "Energy Consumption Meter Consecutive Report Interval", description: "When the device is connected to the gateway, it periodically sends reports from its energy consumption sensor even if there is no change in the value. This parameter defines the interval between consecutive reports of real time and cumulative energy consumption data to the gateway",
+     name: "Selected Energy Report Interval in minutes", paramNum: 141, size: 1, default: 10, type: "number", min: 1 , max: 120, unit: "min"],
+
+    [title: "Control Energy Meter Report", description: "This Parameter determines if the change in the energy meter will result in a report being sent to the gateway. Note: When the device is turning ON, the consumption data will be sent to the gateway once, even if the report is disabled.",
+     name: "Sending Energy Meter Reports", options: [
+                0: "Disabled",
+                1: "Enabled"
+        ], paramNum: 142, size: 1, default: "1", type: "enum"]
+ 
+]}
```

Comment:
Please reformat parameterMap and fix indentations (replace 4 spaces -> 1 tab). 
I mean something like this:
```
	[
		name: "Selected Relay 2 Mode", paramNum: 8, size: 1, default: "0", type: "enum"
		options: [
			0: "NO - Normal Open",
			1: "NC - Normal Close"
		]
	]
```